Week 10/RecipeApp/app/src/main/java/com/janewaitara/movieapp/di/databaseModule.kt

```diff
@@ -0,0 +1,12 @@
+package com.janewaitara.movieapp.di
```

Comment:
Nice job on this and the below modules, you've done a nice job identifying dependencies and separating them into different files.